src/NServiceBus.Core/Pipeline/BehaviorContextStacker.cs

```diff
@@ -8,7 +8,7 @@
     /// <summary>
     /// A stack of <see cref="BehaviorContext"/>s.
     /// </summary>
-    public class BehaviorContextStacker : IDisposable
+    internal class BehaviorContextStacker : IDisposable
```

Comment:
I've had that argument without success

Am Freitag, 24. Juli 2015 schrieb Adam Ralph :

> In src/NServiceBus.Core/Pipeline/BehaviorContextStacker.cs
> <https://github.com/Particular/NServiceBus/pull/2784#discussion_r35399822>
> :
>
> > @@ -8,7 +8,7 @@
> >      /// <summary>
> >      /// A stack of <see cref="BehaviorContext"/>s.
> >      /// </summary>
> > -    public class BehaviorContextStacker : IDisposable
> > +    internal class BehaviorContextStacker : IDisposable
>
> I realise it's not the standard in the Particular org, but I'm [image:
> :+1:] for explicit access modifiers always.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/2784/files#r35399822>.
>